src/current/v23.2/cloud-storage-authentication.md

```diff
@@ -388,6 +392,20 @@ For CockroachDB clusters running in other environments, `implicit` authenticatio
 
 {% include {{ page.version.version }}/backups/external-io-implicit-flag.md %}
 
+<a id="use-implicit-authentication-dedicated">
+### Use implicit authentication with CockroachDB {{ site.data.products.dedicated }}
+
+For a CockroachDB {{ site.data.products.dedicated }} cluster, you can either [use implicit authentication with your own service account](#use-implicit-authentication-with-your-own-service-account) or you can follow these instructions to use implicit authentication with the service account that CockroachDB {{ site.data.products.cloud }} automatically creates and manages for each CockroachDB {{ site.data.products.dedicated }} cluster. This service account is named according to the pattern `crl-dr-store-user-{CLUSTER_ID}`, where `{CLUSTER_ID}` is the ID of the cluster, and is automatically granted the permission to write to the cluster's storage. One benefit of this approach is that no additional credentials needs to be added to the cluster.
+
+For example, to initiate a manual backup on a CockroachDB {{ site.data.products.dedicated }} cluster using implicit auth with the `crl-dr-store-user-{CLUSTER_ID}` service account:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BACKUP INTO 'gs://{bucket name}/{path}?&AUTH=implicit&IAM_ROLE=crl-dr-store-user-{CLUSTER_ID}'
```

Comment:
This flow is also possible with AWS - customers can give `arn:aws:iam::{AWS account ID}:role/crl-dr-store-user-{cluster ID suffix}` direct permissions on an S3 bucket. Is that something to add as part of this PR, or a different one?